resources/knative-eventing/charts/knative-eventing/templates/pre-upgrade-job.yaml

```diff
@@ -36,29 +45,109 @@ data:
   pre-upgrade.sh: |
     #!/usr/bin/env bash
     set -eu
+    RED='\033[0;31m'
```

Comment:
are those Color Escape code really required? If the logs are just printed without interpreting them the logs will look quite ugly

Reply:
Removed color coding